### DISCUSSION TITLE
PrimeNG table filters override behavior

### DISCUSSION THREAD

**Discussion by GBaldoncini** [primefaces/community#1577] [PrimeNG]
Good morning,
I'm using PrimeNG with Angular, and I would like to override the behavior of the filters, specifically the 'not equal' filter for the table. While all other filters are applied only after pressing the enter key, the 'not equal' filter is applied immediately after being selected, causing the table to empty. How can I make the 'not equal' filter be applied in the same way as the others, only after pressing the enter key?
This is the stackblitz [link](https://stackblitz.com/edit/stackblitz-starters-3jwdhi?file=src%2Fmain.ts)

Thank you

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for reporting the issue, we'll try to address it in v17.15.0 https://github.com/primefaces/primeng/issues/15283
